app/js/components/orderEntry/Paginate.jsx

```diff
@@ -2,6 +2,21 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 
+const getNextPageNumber = (nextPageUrl, totalPage) => {
+  const ordersOnPage = 10;
+  if (nextPageUrl) {
+    const pageNumber = Math.floor(nextPageUrl.split('=')[nextPageUrl.split('=').length - 1] / ordersOnPage); 
+    return pageNumber;
+  }
+  return Math.ceil(totalPage / ordersOnPage)
```

Comment:
What does total page mean?
What does orders on page mean?

Reply:
`totalPage` means the total number of pages. 
`ordersOnPage` means the maximum number of Orders a page should have.

Reply:
Will the maintainers of this code always have to first ask you what theses variables mean?

Reply:
This has been corrected. 
for `totalPage`,  I used `totalNumberOfOrders`
and for `ordersOnPage`, I used `maxNumberOfOrdersOnAPage`

Reply:
What does total number of orders mean?

Reply:
`totalNumberOfOrders` is the number of lab orders and drug orders on the page.

Reply:
What is the difference between total number of orders and total number of orders on the page?

Reply:
@dkayiwa, I do not understand your question.

Reply:
What is the difference between "total number of orders" and "total number of orders on the page"?

Reply:
`totalNumberOfOrders`: is the total of all the drug and lab orders coming from the server. 
`maxNumberOfOrdersOnAPage`: is the maximum numbers of drug and lab orders which can stay on a page. which is set to 10.  And this can be adjusted to any number according to preference.

Reply:
So what did you mean around 29 minutes ago by this? "totalNumberOfOrders is the number of lab orders and drug orders on the page."

Reply:
that was an error. 
It's not for the page.
It's the overall number of orders (lab and drug) coming from the server.